website/docs/getting-started/configure.md

```diff
@@ -4,4 +4,82 @@ sidebar_position: 6
 ---
 # Configuring odo global settings
 
-This doc explains everything in `~/.odo/preference.yaml` file.
\ No newline at end of file
+The global settings for odo can be found in `preference.yaml` file; which is located by default in the `.odo` directory of the user's HOME directory.
+
+Example -
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+defaultValue="Linux"
+values={[
+{label: 'Linux', value: 'linux'},
+{label: 'Windows', value: 'windows'},
+{label: 'Mac', value: 'mac'},
+]}>
+<TabItem value="linux">/home/user/foo/myProject/.odo/preference.yaml</TabItem>
+<TabItem value="windows">C:\\myProject\.odo\preference.yaml</TabItem>
```

Comment:
Is it really under `C:\myProject` directory? I would be surprised if so. It should be in a user's home dir.

Reply:
No, I was just going for a different path for some reason. You're right, the default path for preference.yaml is different.